clouddriver-oracle-bmcs/src/main/groovy/com/netflix/spinnaker/clouddriver/oraclebmcs/model/OracleBMCSSecurityGroup.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2017 Oracle America, Inc.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+package com.netflix.spinnaker.clouddriver.oraclebmcs.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.netflix.spinnaker.clouddriver.model.SecurityGroup
+import com.netflix.spinnaker.clouddriver.model.SecurityGroupSummary
+import com.netflix.spinnaker.clouddriver.model.securitygroups.Rule
+import com.netflix.spinnaker.clouddriver.oraclebmcs.OracleBMCSCloudProvider
+import groovy.transform.Immutable
+
+@Immutable
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+class OracleBMCSSecurityGroup implements SecurityGroup {
+  final String type = OracleBMCSCloudProvider.ID
+  final String cloudProvider = OracleBMCSCloudProvider.ID
+  final String id
+  final String name
+  final String description
+  final String application
+  final String accountName
+  final String region
+  final String network
+  final List<String> targetTags
```

Comment:
If `targetTags` isn't a thing on OBMC, feel free to drop it. On GCE it is used for identifying target vm's.